modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/entity/ChapterIndices.kt

```diff
@@ -0,0 +1,12 @@
+package au.com.shiftyjelly.pocketcasts.models.entity
+
+data class ChapterIndices(private val delegate: List<Int> = emptyList()) : List<Int> by delegate {
+    companion object {
```

Comment:
I don't have a strong opinion on this but I wonder if we shouldn't move this logic from the companion object directly to the database converter. I don't think we should expect chapters to be strings anywhere outside of the context of the database. But like I said, I don't have strong feelings about it.

Reply:
Np, I updated it in https://github.com/Automattic/pocket-casts-android/pull/1856/commits/ddfe1bd879af3da85d958b41ccd3a580557a0ee2